pallets/parachain-staking/src/lib.rs

```diff
@@ -715,6 +750,42 @@ pub mod pallet {
 			<InflationConfig<T>>::put(config);
 			Ok(().into())
 		}
+		/// Set the account that will hold funds set aside for parachain bond
+		#[pallet::weight(0)]
+		pub fn set_parachain_bond_account(
+			origin: OriginFor<T>,
+			new: T::AccountId,
+		) -> DispatchResultWithPostInfo {
+			frame_system::ensure_root(origin)?;
+			let ParachainBondConfig {
+				account: old,
+				percent: pct,
+			} = <ParachainBondInfo<T>>::get();
+			<ParachainBondInfo<T>>::put(ParachainBondConfig {
+				account: new.clone(),
+				percent: pct,
+			});
+			Self::deposit_event(Event::ParachainBondAccountSet(old, new));
+			Ok(().into())
+		}
```

Comment:
> Will the parachain bond reserve account defined at genesis or hardcoded ?

@crystalin  It is not set at genesis and not hardcoded -- It can be set at any time by root via `set_parachain_bond_account` extrinsic.

Reply:
Sounds good +1

Reply:
And what happens to funds that would have gone into the bond account before the account is set?

Reply:
If the Account.is_default() (which is what it is if it isn't set), then the 30% is distributed to the collators/nominators. So the inflation rate is the same as it would otherwise be.